assets/js/alerts.js

```diff
@@ -88,6 +88,34 @@ jQuery(function () {
         $(row).addClass('active-alert')
       }
     },
+    ajaxErrorHandler: function (jqXHR, textStatus, errorThrown) {
```

Comment:
This is fine but as a separate PR just thinking, maybe this could be in a Nunjucks macro which is included in each data tables page njk (can use the proper macro for alerts then) and defaults to hidden, this would then just remove/add a class to show the message? Could also then just change the text? 🤔